lib/class-wp-theme-json-resolver.php

```diff
@@ -180,6 +180,10 @@ private function get_theme_origin( $theme_support_data = array() ) {
 	 * @return array Custom Post Type for the user's origin config.
 	 */
 	private static function get_user_data_from_custom_post_type( $should_create_cpt = false, $post_status_filter = array( 'publish' ) ) {
+		if ( ! gutenberg_experimental_global_styles_has_theme_json_support() ) {
```

Comment:
The same comment as in get_origin, it seems this function should always return the user custom_post_type and the checks should not be here.

I guess the checks should be in gutenberg_experimental_global_styles_enqueue_assets and gutenberg_experimental_global_styles_settings?